pkg/config/install_requirements_test.go

```diff
@@ -356,3 +357,53 @@ func TestHelmfileRequirementValues(t *testing.T) {
 	assert.FileExists(t, file)
 	assert.FileExists(t, valuesFile, "file %s should exist", valuesFile)
 }
+
+func Test_LoadRequirementsConfig(t *testing.T) {
+	t.Parallel()
+
+	var gitPublicTests = []struct {
+		requirementsPath   string
+		createRequirements bool
+		loadSuccessful     bool
+	}{
+		{"a", false, false},
+		{"a/b", false, false},
+		{"a/b/c", false, false},
+		{"e", true, true},
+		{"e/f", true, true},
+		{"e/f/g", true, false},
+	}
+
+	for _, testCase := range gitPublicTests {
+		t.Run(testCase.requirementsPath, func(t *testing.T) {
+			dir, err := ioutil.TempDir("", "jx-test-load-requirements-config")
+			assert.NoError(t, err, "failed to create tmp directory")
+			defer func() {
+				_ = os.RemoveAll(dir)
+			}()
+
+			testPath := filepath.Join(dir, testCase.requirementsPath)
+			err = os.MkdirAll(testPath, 0700)
+			assert.NoError(t, err, "unable to create test path %s", testPath)
```

Comment:
NIT: `require.NoError`

Reply:
Interesting, what's the difference? Is there a semantic difference? I think _require_ crossed my path at some stage, but I usually use assert. Well, personally I prefer to write a Gingko test if start a new test ;-)